kakmerge

```diff
@@ -285,6 +311,14 @@ main() {
             exit 1;;
     esac
 
+    case "${multiplexer}" in
+        ""|"tmux"|"kitty");;
+        *)
+            log_error "Unsupported multiplexer: ${multiplexer}"
+            usage
+            exit 1;;
```

Comment:
Could you just use `log_fatal` here, instead of the `usage` and `exit 1` combo?

Also, document the option in `usage()`:

```
usage() {
    log_msg "Usage: $0 [-h] [-c <cmd>] [data]...

Options:
  -m {tmux,kitty} Use a specific terminal multiplexer (default: tmux)
  -c <cmd>        Execute command <cmd> (default: merge), can be one of: merge, abort, done, pick
```

Reply:
`log_fatal` doesn't print `usage`, I thought that's the reason you don't use it here when validating `$command`. Should I remove the call to `usage` completely?

Good catch with documentation 👍

Reply:
As this is mostly a non-interactive tool, I don't think we need to go the extra mile and print the usage whenever an error occurs. Remove `usage` and let `log_fatal` call `exit`, it will keep the code short.

Reply:
Your wish is my command 🙂